frontend/app/module/input/page/InputHomePage.tsx

```diff
@@ -23,6 +30,18 @@ export function InputHomePage() {
           <WorkStationNumber>16</WorkStationNumber>
         </section>
       </header>
+      {/* For demo purposes we just check if _any_ polling station is completed */}
+      {statuses.every((s) => s.status === "Complete") && (
```

Comment:
This comment doesn't match the implementation

Reply:
Whoops, I updated the implementation, but not the comment

Reply:
[965a753](https://github.com/kiesraad/abacus/pull/238/commits/965a753092b54ae7b8a1bd3c81ff16250068ff66)